tracing/tests/event.rs

```diff
@@ -8,27 +8,25 @@
 
 #[macro_use]
 extern crate tracing;
-mod support;
-
-use self::support::*;
 
 use tracing::{
     field::{debug, display},
     subscriber::with_default,
     Level,
 };
+use tracing_test::*;
 
 macro_rules! event_without_message {
     ($name:ident: $e:expr) => {
         #[test]
         fn $name() {
-            let (subscriber, handle) = subscriber::mock()
+            let (subscriber, handle) = subscriber::expect()
                 .event(
-                    event::mock().with_fields(
-                        field::mock("answer")
+                    event().with_fields(
+                        field("answer")
                             .with_value(&42)
                             .and(
-                                field::mock("to_question")
+                                field("to_question")
                                     .with_value(&"life, the universe, and everything"),
                             )
                             .only(),
```

Comment:
it might be worth reworking this API a bit to reduce the rightward drift from multiple fields, like
```rust
event()
   .with_field(field("answer").with_value(&42))
   .with_field(field('to_question").with_value(&"life, the universe, and everything"))
   .only()
```
but, that would be more than a simple mechanical change, so take it or leave it.

We could also do this in a follow-up PR, but we'd want to make the change before releasing the crate, because it would be a breaking change.